src/memory/client/MemoryBrowser.tsx

```diff
@@ -77,21 +77,50 @@ interface State {
   bytesPerRow: number;
   bytesPerGroup: number;
   endianness: 'le' | 'be';
+  childrenNames: { id: number; name: string }[];
 }
 
 export class MemoryBrowser extends React.Component<Props, State> {
   private addressReq = '';
   private lengthReq = '512';
+  private childReq = 0;
+  public static firstTime: boolean = true;
 
   constructor(props: Props) {
     super(props);
     this.state = {
       bytesPerRow: 32,
       bytesPerGroup: 8,
       endianness: 'le',
+      childrenNames: [],
     };
   }
 
+  async getAvailableChilds() {
```

Comment:
Can you rename this `getAvailableChildren`?

Reply:
I have updated it.